services/api/pepc_boost_test.go

```diff
@@ -633,11 +823,96 @@ func TestSubmitTobTxsInSequence(t *testing.T) {
 			network:            common.EthNetworkCustom,
 			nextSentIsHigher:   false,
 		},
+		{
+			description: "goerli first set of txs is higher",
+			firstTobTxs: []*gethtypes.Transaction{
+				gethtypes.NewTx(&gethtypes.LegacyTx{
+					Nonce:    validEthUsdcTx.Nonce(),
+					GasPrice: validEthUsdcTx.GasPrice(),
+					Gas:      validEthUsdcTx.Gas(),
+					To:       validEthUsdcTx.To(),
+					Value:    validEthUsdcTx.Value(),
+					Data:     validEthUsdcTx.Data(),
+				}),
+				gethtypes.NewTx(&gethtypes.LegacyTx{
+					Nonce:    2,
+					GasPrice: big.NewInt(2),
+					Gas:      2,
+					To:       &backend.relay.relayerPayoutAddress,
+					Value:    big.NewInt(2),
+					Data:     []byte(""),
+				}),
+			},
+			secondTobTxs: []*gethtypes.Transaction{
+				gethtypes.NewTx(&gethtypes.LegacyTx{
+					Nonce:    validEthUsdcTx.Nonce(),
+					GasPrice: validEthUsdcTx.GasPrice(),
+					Gas:      validEthUsdcTx.Gas(),
+					To:       validEthUsdcTx.To(),
+					Value:    validEthUsdcTx.Value(),
+					Data:     validEthUsdcTx.Data(),
+				}),
+				gethtypes.NewTx(&gethtypes.LegacyTx{
+					Nonce:    4,
+					GasPrice: big.NewInt(5),
+					Gas:      12,
+					To:       &backend.relay.relayerPayoutAddress,
+					Value:    big.NewInt(1),
+					Data:     []byte(""),
+				}),
+			},
+			firstTobTxsTraces:  validEthUsdcTxTrace,
+			secondTobTxsTraces: validEthUsdcTxTrace,
+			network:            common.EthNetworkGoerli,
+			nextSentIsHigher:   false,
+		},
+		{
+			description: "goerli second set of tob txs is higher",
+			firstTobTxs: []*gethtypes.Transaction{
+				gethtypes.NewTx(&gethtypes.LegacyTx{
+					Nonce:    validEthUsdcTx.Nonce(),
+					GasPrice: validEthUsdcTx.GasPrice(),
+					Gas:      validEthUsdcTx.Gas(),
+					To:       validEthUsdcTx.To(),
+					Value:    validEthUsdcTx.Value(),
+					Data:     validEthUsdcTx.Data(),
+				}),
+				gethtypes.NewTx(&gethtypes.LegacyTx{
+					Nonce:    2,
+					GasPrice: big.NewInt(2),
+					Gas:      2,
+					To:       &backend.relay.relayerPayoutAddress,
+					Value:    big.NewInt(2),
+					Data:     []byte(""),
+				}),
+			},
+			secondTobTxs: []*gethtypes.Transaction{
+				gethtypes.NewTx(&gethtypes.LegacyTx{
+					Nonce:    validEthUsdcTx.Nonce(),
+					GasPrice: validEthUsdcTx.GasPrice(),
+					Gas:      validEthUsdcTx.Gas(),
+					To:       validEthUsdcTx.To(),
+					Value:    validEthUsdcTx.Value(),
+					Data:     validEthUsdcTx.Data(),
+				}),
+				gethtypes.NewTx(&gethtypes.LegacyTx{
+					Nonce:    4,
+					GasPrice: big.NewInt(5),
+					Gas:      12,
+					To:       &backend.relay.relayerPayoutAddress,
+					Value:    big.NewInt(10),
+					Data:     []byte(""),
+				}),
+			},
+			firstTobTxsTraces:  validEthUsdcTxTrace,
+			secondTobTxsTraces: validEthUsdcTxTrace,
+			network:            common.EthNetworkGoerli,
```

Comment:
why do these need to be specific to goerli ? isnt this generic enough for all networks?